pkg/querier/frontend/query_range.go

```diff
@@ -91,6 +92,15 @@ func (q QueryRangeRequest) toHTTPRequest(ctx context.Context) (*http.Request, er
 	return req.WithContext(ctx), nil
 }
 
+func (q QueryRangeRequest) logToSpan(ctx context.Context) {
+	if span := opentracing.SpanFromContext(ctx); span != nil {
+		span.LogFields(otlog.String("query", q.Query))
+		span.LogFields(otlog.String("start", timestamp.Time(q.Start).String()))
+		span.LogFields(otlog.String("end", timestamp.Time(q.End).String()))
+		span.LogFields(otlog.Int64("step (ms)", q.Step))
```

Comment:
Nit: should this be one call to LogFields (its variadic)?  Not sure what the best practices here are.